services/order.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from django.db import transaction
+from db.models import Order, Ticket
+from django.core.exceptions import ValidationError
+from django.contrib.auth import get_user_model
+from datetime import datetime
+
+User = get_user_model()
+
+
+def create_order(tickets: List[dict],
+                 username: str,
+                 date: [str] = None) -> None:
+
+    try:
+        user = User.objects.get(username=username)
+    except User.DoesNotExist:
+        raise ValidationError("User does not exist")
+
+    if date:
+        created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+    else:
+        created_at = datetime.now()
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user, created_at=created_at)
+        for ticket in tickets:
+            Ticket.objects.create(
+                order=order,
+                movie_session_id=ticket["movie_session"],
+                row=ticket["row"],
+                seat=ticket["seat"]
```

Comment:
Consider validating the `tickets` list to ensure each dictionary contains valid `movie_session`, `row`, and `seat` keys with appropriate values before creating `Ticket` objects. This will prevent potential errors during ticket creation.